plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -294,6 +305,19 @@ BOOL CALLBACK ConfigProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
                     EndDialog(hWnd, 0);
                     break;
 
+                case IDC_OUTPUT:
+                    if (wmEvent == CBN_SELCHANGE) {
+                    const int module = (int)SendMessage(GetDlgItem(hWnd, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
```

Comment:
I don't see anything wrong with the formatting (that I know of) but i'll check in a bit when I'm not on a mobile screen.